mne/viz/epochs.py

```diff
@@ -374,9 +374,8 @@ def pair_and_combine(data):
             this_data = combine(
                 data[:, picks_, :])[:, np.newaxis, :]
         info = pick_info(epochs.info, [picks_[0]], copy=True)
+        info['projs'] = []
         these_epochs = EpochsArray(this_data.copy(), info, tmin=tmin)
-        d = these_epochs.get_data()  # Why is this necessary?
-        d[:] = this_data  # Without this, the data is all-zeros!
```

Comment:
FYI @jona-sassenhagen this happens because of the projection vectors -- the warning about using 1 channel being dangerous was the hint :)

Reply:
Tbh I still don't understand projections.

Is there a better way to do this?

Reply:
This PR implements a cleaner way

Reply:
Oh great, I see. Thanks.